platform/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@metamask/sdk-react-ui": "^0.11.2",
-    "next": "14.0.1",
+    "next": "^14.0.1",
```

Comment:
Please do not change a version of existing package. That effects all developers in team and some packages may have conflicts with new version.